### PR TITLE
Preserve data in LocalStorage

### PR DESCRIPTION
**Pull request by HtetNaing0814**
## Implement the following interactions:
- When the user changes the content of any input field, the data is saved to the local storage.
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

## Used the following data model:
- Created a single JavaScript object with all the data from the entire form and save it in local storage.

P.S. My partner just got removed from the group just today. He took a one-week leave. So I am alone. That's why the repo has only my commits. Please kindly do understand the situation. Thank u so much.

### CODE REVIEW

**Review by BenMukebo — COMMENTED** [review]
## STATUS: INVALID ❌ 

Hi  Team,

I can't miss felicitating your efforts and energy, you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project, that doesn't have commits from all members in your pair programming group. Kindly note that this is a pair programming activity and all the members should commit to the project.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@BenMukebo) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit.

**Comment by KDlamini** [comment]
> My partner just got removed from the group. He took a one-week leave. So I am alone. That's why the repo has only my commits. Please kindly do understand the situation. Thank u so much.

@HtetNaing0814 Alright, thank you for letting us know. 👍🏾

File: script.js [hunk 1]
```diff
@@ -120,10 +124,24 @@ const EMAIL_REQUIRED = 'Please enter your email';
...
 
 form.addEventListener('submit', (event) => {
+  const formObj = {
+    nameID: nameID.value,
+    email: email.value,
+    textarea: textarea.value,
+  };
+
+  window.localStorage.setItem('formObj', JSON.stringify(formObj));
+
```
[the remark this answers is not in the document]
Comment (HtetNaing0814) [reply script.js:134]: Does that mean data should be saved to local storage even before submission? Even when the user changed the inputs, data should be saved to local storage? Like that, brother?

[the remark this answers is not in the document]
Comment (KDlamini) [reply script.js:134]: @HtetNaing0814  Yes, data should be saved before submitting. The data should also be updated if the user changes the inputs.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @HtetNaing0814      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**